### PR TITLE
feat(containers): add Redirect + HomeListView components

### PR DESCRIPTION
**Pull request by jmfrancois**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

To build a CMF app you need at least one view.
You also need to redirect from / to the real homepage path.

No components provide this in the current stack.

**What is the new behavior?**

Let's have an homepage view and redicrect component.

**Other information**:

screen shot 2017-02-08 at 16 55 02

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://132.talend.surge.sh)

File: packages/containers/src/HomeListView/HomeListView.component.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import React, { PropTypes } from 'react';
+import { api } from 'react-cmf';
+import { Layout } from 'react-talend-components';
+
+import AppHeaderBar from '../AppHeaderBar';
+import List from '../List';
+import SidePanel from '../SidePanel';
+
+function HomeListView(props) {
```
Comment (acateland) [inline packages/containers/src/HomeListView/HomeListView.component.js]: a bit of destructuring here will help the user to discover the component interface

File: packages/containers/src/HomeListView/HomeListView.container.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+			this.props.dispatch(
+				api.action.getActionCreatorFunction(
+					this.context, this.props.didMountActionCreator
+				)()
+			);
+		}
+	}
+
+	render() {
+		// const state = (this.props.state || DEFAULT_STATE).toJS();
```
Comment (acateland) [inline packages/containers/src/HomeListView/HomeListView.container.js]: removing comment

File: packages/containers/src/Redirect/Redirect.component.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import { CircularProgress } from 'react-talend-components';
+
+export const REDIRECT_ACTION_TYPE = 'REDIRECT';
+
+/**
+ * @param {object} props react props
+ * @example
+<Redirect name="Hello world"></Redirect>
+ */
+function Redirect(props, context) {
```
Comment (acateland) [inline packages/containers/src/Redirect/Redirect.component.js]: same a bit of destructuring here would be awesome

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://132.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://132.talend.surge.sh)

**Pull request merged**